docs/cues/creating-a-cue.md

```diff
@@ -100,19 +100,23 @@ Other useful things to know about recording cues:
 
 ## Quick Build Cues
 
-Setting <Keys.SoftKey>Record Mode</Keys.SoftKey> to **Quick Build** allows you to build a cue from
+Setting <Keys.SoftKey>Record Mode</Keys.SoftKey> to **Quick Build** allows you to build a cue, chase or cue list from
 existing playbacks or palettes (you can also use the
 [Include function](editing-cues.md#using-parts-of-existing-cues-the-include-function) to
 do this).
 
-After setting the record mode to Quick Build, the console will wait for
-you to select playbacks or palettes.
+1.  Press Record.
+2.  Press <Keys.SoftKey>Record Mode</Keys.SoftKey> and set the mode to Quick Build.
+3.  Select whether you want to create a Memory, Chase, Cue List or Timeline using the softkey options.
```

Comment:
Quickbuild doesn't do anything special with timeline so maybe you don't want to mention it here.